### PR TITLE
Create BAL-INTEREST-RATE-UPDATES.md

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Add AIP to update BAL interest rates across different pools.
Updates Polygon v3 borrow cap for BAL.

### CODE REVIEW

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: [ARC] BAL Interest Rate Curve Upgrade
+discussion: https://governance.aave.com/t/arfc-bal-interest-rate-curve-upgrade/10484
+shortDescription: Upgrade the BAL interest rate curve across Ethereum v2, Polygon v3, and Polygon v2 Liquidity Pools
+author: [Llama](https://twitter.com/llama) [DeFi_Consulting](https://twitter.com/DeFi_Consulting) [scottincrypto](https://twitter.com/scottincrypto)
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:5]: i think there is currently no governance ui handling links here

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+created: 2023-02-01
+—
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:7]:
```suggestion
---
```

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+title: [ARC] BAL Interest Rate Curve Upgrade
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:2]: when it's an aip, it's no longer an "aave request for comments" i assume
```suggestion
title: BAL Interest Rate Curve Upgrades
```

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+discussion: https://governance.aave.com/t/arfc-bal-interest-rate-curve-upgrade/10484
+shortDescription: Upgrade the BAL interest rate curve across Ethereum v2, Polygon v3, and Polygon v2 Liquidity Pools
+author: [Llama](https://twitter.com/llama) [DeFi_Consulting](https://twitter.com/DeFi_Consulting) [scottincrypto](https://twitter.com/scottincrypto)
+created: 2023-02-01
+—
+
+# Summary
+
+@Llama presents a proposal to amend BAL interest rate parameters on the Aave Ethereum v2, Polygon v3 and Polygon v2 Liquidity Pools.
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:11]: here you can use links

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+With Aave receiving 20% of all fee revenue, this will lead to increased BAL nominated revenue. 
+
+In addition to the interest rate curve changes, this proposal will also increase the BorrowCap from 156.3K BAL to 256.140K BAL. Revising the BorrowCap higher will enable users to benefit from the increased the Uoptimal parameter. Do note, the BorrowCap figure was provided by Chaos Labs. 
+
+# Specification
+
+The below table shows the current and proposed changes to the BAL Interest Rate on Ethereum v2, Polygon v2 and Polygon v3.
+
+|Parameter|Current (%)|Proposed (%)|
+| --- | --- | --- |
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:42]:
```suggestion
| --- | ---: | ---: |
```
you can right align numbers

File: content/aips/BAL-INTEREST-RATE-UPDATES.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Proposal Payload deployed address can be found [here] (https://etherscan.io/address/0x3b6532efb7a60711f8f3fa77a589a726c836f4cf)
+Proposal Payload for Polygon deployed address can be found [here] (https://polygonscan.com/address/0x98bc9dfa3cecb37f1bdeadc6e774d39082756b19)
+
+Interest Rate Strategy deployed address can be found [here](https://etherscan.io/address/0x04c28D6fE897859153eA753f986cc249Bf064f71)
+Interest Rate Strategy deployed address for Polygon v2 can be found [here](https://polygonscan.com/address/0x80cb7e9E015C5331bF34e06de62443d070FD6654)
+Interest Rate Strategy deployed address for Polygon v3 can be found [here](https://polygonscan.com/address/0x4b8D3277d49E114C8F2D6E0B2eD310e29226fe16)
+
+# Copyright
+
+Copyright and related rights waived via [CC0\(https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (sakulstra) [inline content/aips/BAL-INTEREST-RATE-UPDATES.md:75]:
```suggestion
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

File: content/aips/BAL-INTEREST-RATE-UPDATES.md (hunk 5 above)
Comment (efecarranza) [reply content/aips/BAL-INTEREST-RATE-UPDATES.md:42]: This is cool

**Pull request merged**